cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -428,11 +448,26 @@ def test_datastore_relationships():
             1.
             2.
             3. Operations should be performed successfully. It should not give unexpected error.
-
-    Bugzilla:
-        1729953
     """
-    pass
+    view = navigate_to(testing_vm.datastore, "ManagedVMs")
```

Comment:
Do you really need to provision a VM in order to navigate to its datastore's ManagedVMs view? I understand that this is a part of BZ steps-to-reproduce, but can you not just add a provider, select a datastore, navigate to the ManagedVMs page and test?

Reply:
Yes. This is possible:
```
view = navigate_to(appliance.collections.datastores.all()[0], "ManagedVMs")
```
- But in this test case, I need to have one VM on which I can perform relationships.